modules/tinymce/src/core/main/ts/api/util/Delay.ts

```diff
@@ -97,7 +97,10 @@ const debounce = <T extends (...args: any[]) => any>(callback: T, time?: number)
 const Delay: Delay = {
   /**
    * Requests an animation frame and fallbacks to a timeout on older browsers.
+   * <br>
+   * <em>`requestAnimationFrame` has been deprecated and marked for removal in TinyMCE 6.0.</em>
```

Comment:
```suggestion
   * <em><code>requestAnimationFrame</code> has been deprecated and marked for removal in TinyMCE 6.0. Use the native browser <code>requestAnimationFrame</code> API instead.</em>
```
Same fixes as previously and we should also mention the replacement here.